xsd/netex_part_1/part1_ifopt/netex_ifopt_equipmentAccess_version.xsd

```diff
@@ -363,14 +368,24 @@ Rail transport, Roads and Road transport
 			</xsd:element>
 			<xsd:element name="StepHeight" type="LengthType" minOccurs="0">
 				<xsd:annotation>
-					<xsd:documentation>Depth of an individual step.</xsd:documentation>
+					<xsd:documentation>Depth of an individual step, in metres rounded to the nearest cm.</xsd:documentation>
```

Comment:
Still is still an issue, I think this should be in SI_METER (or better just LengthType), and not in cm.